tgui/packages/tgui/interfaces/Manufacturer/collapsibleWireMenu.tsx

```diff
@@ -0,0 +1,71 @@
+import { useBackend } from "../../backend";
+import { Button, Divider, LabeledList, Section } from "../../components";
+import { ManufacturerData } from "./type";
+
+const is_set = (bits, bit) => bits & (1 << bit);
```

Comment:
Also done somewhere else (Bombsim?). Move into a common util.

Reply:
Sounds good, I made bombsim and it's definitely there so

Reply:
done